st2common/st2common/models/db/execution.py

```diff
@@ -111,6 +111,19 @@ def mask_secrets(self, value):
             liveaction['parameters'] = mask_secret_parameters(parameters=liveaction['parameters'],
                                                               secret_parameters=secret_parameters)
 
+            if 'inquiry.respond' in liveaction.get('action', ''):
```

Comment:
Maybe we should just do direct equal check for action name to be on the safe side?

I know it's very unlikely someone will name their action `packname.inquiry.responst.foo`, but still :)

Reply:
Ack, when I started tinkering with this, I couldn't remember if the pack name was included at this point so I did this to get a PoC running and neglected to change it back. Done in d032480ca42a1600584dc9694546ee8aa0767b86